<?xml version="1.0"?>
<launch>
	<!-- Semantic Segmentation node  -->
	<node type="image_bbox_to_3d_stereo.py" name="imageBBoxTo3D_Stereo" pkg="image_boundingbox_to_3d" respawn="true" output="screen">
		<param name="topicCamImage" value="/Multisense/left/image_rect_color" />
		<param name="topicCamInfo" value="/Multisense/left/image_rect_color/camera_info" />
		<param name="topicDepth" value="/Multisense/depth" />
		<param name="topicBBoxIn" value="/yolo/BBox/Multisense" />
		<param name="topicBBoxOut" value="/yolo/BBox3d/Multisense" />
		<param name="topicVisualizeOut" value="/yolo/BBox3dImage/Multisense" />
		<param name="baseFrameId" value="velodyne" />

		<param name="estDistanceMethod" value="1" /> <!-- Select either 0: mean, 1: median, 2: 10%-percentile -->
		<param name="visualizeBoundingboxes" value="False" /> <!-- Select either False: No visualization topic is published, True: A image for visualizing bounding boxes are published  -->
	</node>
</launch>
